src/index.html

```diff
@@ -17,16 +17,32 @@
     />
   </head>
   <body>
-    <input
-      type="text"
-      data-qa="keypress"
-      placeholder="Try “Los Angeles“"
-    />
+    <form
+      action="#"
+      method="get"
+      class="searchBar searchBar--big"
+      data-qa="big"
+    >
+      <input
+        class="searchBar__input searchBar__input--top"
+        type="text"
+        data-qa="keypress"
+        placeholder="Try “Los Angeles“"
+      />
+    </form>
```

Comment:
Your form lacks a submit button. Users should have a way to submit the form without having to rely on hitting the Enter key.